tests/maxwell_solver_test.py

```diff
@@ -13,3 +15,109 @@ def test_1d_uniform():
     true_phi = -2 * np.pi * (ms.grid**2 - 1)
     err = np.abs(true_phi - phi)
     np.testing.assert_allclose(err, np.zeros(len(err)), atol=1e-9)
+
+
+def test_2d_A():
+    ms = MaxwellSolver2D()
+    A = ms.set_A(4)
+
+    true_A = np.array([[-4, 1, 1, 0], [1, -4, 0, 1], [1, 0, -4, 1], [0, 1, 1, -4]])
+    np.testing.assert_allclose(A, true_A, atol=1e-9)
+
+
+def test_2d_boundary():
+    pass
```

Comment:
What is this meant to do?

Reply:
Ah right, I should have a test for setting the boundary conditions correctly.